include/sparrow/utils/nullable.hpp

```diff
@@ -697,6 +716,24 @@ namespace sparrow
     {
         return nullable<T, B>(std::forward<T>(value), std::forward<B>(flag));
     }
+
+    /***********************************
+     * nullable_variant implementation *
+     ***********************************/
+
+    template <class... T>
```

Comment:
Note that for these size of implementations, it would have been reasonable to have the functions definitions in the class definition.